src/components/label/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, h, Prop } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-label",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyLabel {
+	@Prop() color: string
+	@Prop() name: string
+	@Prop() description: string
+	@Prop({ reflect: true }) shape: "rectangle" | "rounded"
+
+	render() {
+		return (
+			<div
```

Comment:
I think you can set this directly on the `<Host>` object, instead of adding an extra `div`.

Reply:
\+